react/src/pages/MainMenu/MainMenu.tsx

```diff
@@ -11,35 +9,8 @@ import designsafeLogo from '@hazmapper/assets/designsafe.svg';
 import nheriLogo from '@hazmapper/assets/nheri.png';
 
 const MainMenu = () => {
-  const location = useLocation();
```

Comment:
@sophia-massie , moving the contextholder into a context to the top-level `react/src/index.tsx` did the trick with getting the notification to appear after a navigation.